packages/adapters-library/src/scripts/buildMetadata.ts

```diff
@@ -70,18 +73,32 @@ export function buildMetadata(
               continue
             }
 
-            const { metadata, fileDetails } = (await adapter.buildMetadata(
-              true,
-            )) as {
-              metadata: Json
-              fileDetails: {
-                protocolId: Protocol
-                productId: string
-                chainId: Chain
-                fileKey: string
-              }
+            const metadataDetails = (await adapter
+              .buildMetadata(true)
+              .catch((e) => {
+                if (!(e instanceof NotImplementedError)) {
+                  throw e
+                }
```

Comment:
Needed so that it ignores building metadata when it throws because that specific chain uses logs.